README.md

```diff
@@ -41,6 +41,50 @@ Of course, `AvatarView` also supports `Source` for loading images plus a few oth
 
 ![AvatarView](./images/avatar-do.png)
 
+## Gesture Pad
+
+### Using Gesture Pad
+
+Call InitGesturePad method after Xamarin.Forms.Init();
```

Comment:
Two questions

1) Can we avoid invoking InitGesturePad somehow?
2) Should we rename this control to "SignaturePad"?

Reply:
On most platforms yes. on Tizen and Gtk we have to specify the parent. to be more consistent with all platforms I added InitGesturePad. Also we register Dependency there.
As for the naming, it is up to you. I prefer Gesture, because I understand it as more general term rather then signature

Reply:
Let's collect more opinions then :) 
CC @pictos @jfversluis @sthewissen

Reply:
> Can we avoid invoking InitGesturePad somehow?

I was thinking the same thing. I think Tizen and Gtk will be less used anyway and it will be better to not burden the majority of our users (iOS, Android, UWP) with an Init call. So if we can keep it to just Gtk and Tizen that would be best.

> Naming

Yeah I was thinking about that too. I understand signature and gesture, but I feel both are not covering it all. It's more than a gesture, because it's drawing something and it's not a signature one because it can do more than just signatures :) Also, I'd like to avoid confusion with the SignaturePad that is already out there. DrawingPad? DrawPad?

Reply:
DrawingBoard | DrawBoard | DrawBoardView | DrawingBoardView

Reply:
what about DrawingView

Reply:
DrawingView, or maybe simpler just DrawView. Works for me :)

Reply:
> On most platforms yes. on Tizen and Gtk we have to specify the parent.

Rigjt, we need a parent object to create a native control on tizen, but you don't need to additional Init() for this. Please refer to the [comments ](https://github.com/xamarin/XamarinCommunityToolkit/pull/468#discussion_r509831037)below.